### DISCUSSION TITLE
TFT70 Icon Font Spacing

### DISCUSSION THREAD

**Discussion by vernonbc** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2693] [General]
I recently purchased a TFT70, update the firmware and am using the 'round' icon pack. I noticed in the 'preheat' screen that the temperatures are to far to the right of the icon and almost off the icon. Is there anyway to move the text over to the left so that it is fully inside the icon? 
I have no issues re-compiling the firmware. I just would like to know where I would edit this.
Thank you very much.

**Comment by radek8** [comment]
Can you take a photo of the problem and also take a photo with the classic icons for comparison?
If there is a problem with both sets of icons, the text will need to be moved.
If the problem is only with the round icons, the round icons will need to be adjusted.
Thank you
